### ISSUE TITLE
Training set contains 40 or 50 videos

### ISSUE THREAD

**Issue by Gracezxw** [facebookresearch/AnimalAvatar#4]
You mentioned in your paper that you "select a subset of 50 dog videos". But why are there only 40 sequence numbers in config/keys.py : "subsetcop3d"?

**Comment by RemySabathier** [comment]
The sequence list in "config/keys.py" correspond to the list of asset for which we provide refined masks (compared to original COP3D mask), init pose and sparse keypoint per frame. 
Our eval subset in the paper contains these 40 sequences + 10 additional sequences (we limited the additional data to 40 sequences to limit the filesize).

**Comment by Gracezxw** [comment]
Thanks! By the way, could you please announce the sequence numbers of the 10 additional sequences?

**Comment by RemySabathier** [comment]
Sure, I've added the eval set in config/keys.py

**Comment by Gracezxw** [comment]
Thanks a lot!
